PowerShell/ScubaGear/Modules/Orchestrator.psm1

```diff
@@ -75,6 +75,9 @@ function Invoke-SCuBA {
     .Parameter OutCsvFileName
     The CSV created in the folder created in OutPath that contains the CSV version of the test results.
     Defaults to "ScubaResults".
+    .Parameter OutPlanFileName
```

Comment:
This parameter name seems too generic to describe what this parameter is for. Maybe ActionPlanFileName?

Reply:
I also don't love `OutPlanFileName`, so I'm definitely open to suggestions on it. That said, I'd like to stay consistent with all the other file name parameters, each of which starts with "Out" (see #1354 for a list). Maybe `OutActionPlanFileName`? Certainly more descriptive, I just worry that it might be too long.

Reply:
Good point about tab complete. Alright, I'll change it to `OutActionPlanFileName`.

Reply:
@tkol2022 name change has been pushed.